### DISCUSSION TITLE
PR#450, Please consider merging, --no-static options for runserver

### DISCUSSION THREAD

**Discussion by mangelozzi** [django/daphne#543] [General]
This issue:
https://github.com/django/daphne/issues/449
is solved by PR 450:
https://github.com/django/daphne/pull/450

Which is a really small PR (2 small chunks). Please would the maintainers check it out when you get a chance, I would really appreciate it, it solves issues for people running custom static content servers. The issue is closed, but the PR was never merged. If it is no desirable to merge the PR, then maybe reject it with a reason.

Thank you so much to anyone reading this for your time, hope you have a Merry Christmas!

**Comment by carltongibson** [comment]
@mangelozzi I can look after Christmas.

**Reply by mangelozzi** [reply]
Thanks so much for merging it! This options is especially relevant when using the --no-asgi option and wishing to pass args forward to the original Django runserver command. 

Is there a release with this fix? The latest release is 4.2.0 which is what I currently have before the fix was applied.

**Reply by carltongibson** [reply]
@mangelozzi It was part of 4.2.0 which was released last week.
